gen/calc.py

```diff
@@ -421,7 +425,10 @@ def validate_os_type(os_type):
         },
         'rexray_config_method': {
             'file': {
-                'must': {'rexray_config_contents': calculate_rexray_config_contents},
+                'must': {'rexray_config_contents': calculate_rexray_config_from_file},
+            },
+            'string': {
+                'must': {'rexray_config_contents': calculate_rexray_config_from_string},
             },
```

Comment:
@pyronicide It gives you a preset REX-Ray config that does the right thing if your DC/OS agents are ec2 instances with IAM roles configured. On master it's used in CloudFormation templates, but with this PR I'm moving CF REX-Ray to a CF-specific config.